pipelines/snap-atac/README.md

```diff
@@ -1,10 +1,68 @@
-# Snap-ATAC pipeline Overview
-
-The snap-atac pipeline is a pipeline for processing scATAC-seq datasets, based on the pipeline that is available (in the snap-atac repository)[https://github.com/r3fang/SnapATAC) and [the snap-utils repository](https://github.com/r3fang/SnapATAC).
+| Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| [scATAC 1.1.0 ](scATAC.wdl) | May 18th 2020 | [Elizabeth Kiernan](mailto:ekiernan@broadinstitute.org) | Please file GitHub issues in skylab or contact [Kylee Degatano](mailto:kdegatano@broadinstitute.org) |
+
+- [Overview](#overview)
+- [Introduction](#introduction)
+  * [Quick Start Table](#quick-start-table)
+- [Set-up](#set-up)
+  * [Workflow Installation and Requirements](#workflow-installation-and-requirements)
+  * [Pipeline Inputs](#pipeline-inputs)
+  * [Input File Preparation](#input-file-preparation)
+    + [R1 and R2 FASTQ Preparation](#r1-and-r2-fastq-preparation)
+    + [Input_reference Preparation](#input_reference-preparation)
+- [Workflow Tasks and Tools](#workflow-tasks-and-tools)
+  * [Task Summary](#task-summary)
+    + [AlignPairedEnd](#alignpairedend)
+    + [SnapPre](#snappre)
+      - [Filtering Parameters](#filtering-parameters)
+      - [Snap QC Metrics](#snap-qc-metrics)
+    + [SnapCellByBin](#snapcellbybin)
+    + [MakeCompliantBAM](#makecompliantbam)
+    + [BreakoutSnap](#breakoutsnap)
+- [Outputs](#outputs)
+- [Running on Terra](#running-on-terra)
+- [Versioning](#versioning)
+- [Pipeline Improvements](#pipeline-improvements)
+
+# Overview
+
+<img src="scATAC_diagram.png" width="500">
+
+# Introduction
+
+The scATAC Pipeline was developed by the Broad DSP Pipelines team to process single nucleus ATAC-seq datasets. The pipeline is based on the [SnapATAC pipeline](https://github.com/r3fang/SnapATAC) described by [Fang et al. (2019)](https://www.biorxiv.org/content/10.1101/615179v2.full). Overall, the pipeline uses the python module [SnapTools](https://github.com/r3fang/SnapTools) to align and process paired reads in the form of FASTQ files. It produces an hdf5-structured Snap file that includes a cell-by-bin count matrix at 10 kb resolution. In addition to the Snap file, the final outputs include a GA4GH compliant aligned BAM and QC metrics.
```

Comment:
So as you can tell by how we named the pipeline (s**c**ATAC) we thought up until recently that we were using it for processing single cell data, and learned that it's actually single nucleus. As you've correctly written here. Should we comment something like "Currently the pipeline has been validated on single nucleus atacseq data, but we will evaluate it for single cell atac in the future." so that it's not confusing why it's named that? Open to ideas here.

Reply:
I don't know if in the case of ATAC seq, it makes sense to distinguish single cell vs single nuclei, but I will check in with the CBs to see if they agree? ATAC seq is always going to target chromatin, so my thinking is it's implied that it's nuclear (you have to isolate nuclei to do the assay). Unlike RNA which can be isolated from both cytosol or nucleus, chromosomes (at least the ones with histones) are always nuclear. So single nuclei is single cell in this case.  The 10X technology also isolates nuclei, but calls it single cell (https://www.10xgenomics.com/products/single-cell-atac/). 

So I guess my open questions are how will/does a single cell assay differ from nuclear assay?

Reply:
That's a great observation / clarification, thank you! It seems it's very common in the scATAC world to call it sc even though it's logically sn. So let's do what's expected by users and keep it as sc but as you did, clarify in the documentation that it's processing data from nuclei.

Reply:
Ok, we can update if we find out any assay differences that we should highlight. Thank you!